sdk/eventgrid/eventgrid/package.json

```diff
@@ -3,7 +3,7 @@
   "sdk-type": "client",
   "author": "Microsoft Corporation",
   "description": "An isomorphic client library for the Azure Event Grid service.",
-  "version": "3.0.0-beta.3",
+  "version": "3.0.0-beta.4",
```

Comment:
beta.3 is not yet released... why the increment to 4?

Reply:
Huh, I thought we had shipped 3 but I notice it is not on npmjs.  I moved back to .3 and fixed up the changelog.  I can't actually remember why in October it did not go out.

Thanks for catching this!